JS-Algorithms/challenges.js

```diff
@@ -42,6 +51,25 @@ const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
   // YOUR CODE HERE...
+
+  let COUNTRY_COPY = [...COUNTRY_NAMES];
```

Comment:
Avoid using all uppercase letters for variables naming, as this is usually reserved for constants.

Reply:
Using let for variable declarations must be used when we are expecting to modify the value of the variable.

Here are some resources you can review:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let